app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -944,8 +950,10 @@ aside.quote {
     border-top: 1px solid var(--primary-low);
     padding-top: 0.5em;
   }
-  &.highlighted {
-    animation: background-fade-highlight 2.5s ease-out;
+  @media (prefers-reduced-motion: no-preference) {
+    &.highlighted {
+      animation: background-fade-highlight 2.5s ease-out;
```

Comment:
Does this mean the active post background remains e.g. blue for the entire time?